corehq/apps/locations/fixtures.py

```diff
@@ -141,10 +141,9 @@ def get_all_locations_to_sync(user):
     else:
         all_locations = set()
 
-        user_location = user.sql_location
-        user_locations = set([user_location]) if user_location is not None else set()
```

Comment:
The new substitute `user.sql_locations` returns empty list if there are not assigned locations